cpp_modules/src/mcts/Mcts.cpp

```diff
@@ -164,18 +150,22 @@ void Mcts::performAction(Action action) {
 	// Check if the action matches any explored nodes
 	for (const auto childAction: _rootNode->children | std::views::keys) {
 		if (childAction == action) {
-			_rootState->performAction(action);
+			// _rootState->performAction(action);
```

Comment:
![meme-akakichi-no-eleven-21037739](https://github.com/user-attachments/assets/362ebb02-2985-4d70-8703-4d4dd67d473a)

You know what i'm about to say

Reply:
it is done 😞